@kiva/kv-components/vue/KvContentfulImg.vue

```diff
@@ -172,23 +173,33 @@ export default {
 			default: () => [],
 		},
 	},
-	methods: {
-		buildUrl(image = null, multiplier = 1) {
-			let src = image && image.url ? `${image.url}?` : `${this.contentfulSrc}?`;
-			const width = image ? image.width : this.width;
-			const height = image ? image.height : this.height;
+	setup(props) {
+		const {
+			contentfulSrc,
+			width,
+			height,
+		} = toRefs(props);
+
+		const buildUrl = (image = null, multiplier = 1) => {
+			let src = image && image.url ? `${image.url}?` : `${contentfulSrc}?`;
+			const imgWidth = image ? image.width : width;
+			const imgHeight = image ? image.height : height;
```

Comment:
Because `contentfulSrc`, `width` and `height` are refs, their value needs to be accessed with `.value` when in the setup function, for example:
```
const imgWidth = image ? image.width : width.value;
```

Reply:
This was fixed, thank you!